src/lib.rs

```diff
@@ -9,15 +9,16 @@ SPDX-License-Identifier: GPL-3.0-only OR LGPL-3.0-only
 
 */
 
-pub mod mux;
```

Comment:
What's the purpose of this reordering?

Reply:
It was the default when using the intellij code formatter.

Reply:
Yeah, it happened even in your prior PR, I have fixed that there...